sources/VT100ScreenDelegate.h

```diff
@@ -82,6 +82,9 @@
 // If flag is set, bring the window to front; if not, move to back.
 - (void)screenRaise:(BOOL)flag;
 
+// Set the proxy icon of current session window.
+- (void) setProxyIcon:(NSString *)value;
```

Comment:
remove space after `(void)`. Also obey the naming convention (which is admittedly used inconsistently) to begin delegate method names with `screen`, so this would be `screenSetProxyIcon:`